### ISSUE TITLE
Please help! Cannot reproduce the “higherhrnet_w32_crowdpose_512x512” with the  Crowdpose dataset.

### ISSUE THREAD

**Issue by tonyleala** [open-mmlab/mmpose#950]
The configs, work_dirs, and test results of higherhrnet_w32_crowdpose_512x512 have been uploaded below.

The official results cannot be reproduced. AP is about one point behind the official website results. Please help me check it, thanks!
[file.zip](https://github.com/open-mmlab/mmpose/files/7272863/file.zip)

**Comment by tonyleala** [comment]
The NEW: A problem of reproducing the "res50_coco_512x512" in associative_embedding.

The experimental settings are consistent with the official, but the result of AP is about one point below the official result.

Please help me check it again, thanks!

**Comment by jin-s13** [comment]
A little accuracy fluctuation is possible. 
How many GPUs do you use for training?

**Comment by jin-s13** [comment]
@liqikai9 Could you please help check this issue?

**Comment by jin-s13** [comment]
I am not sure if it will affect the accuracy a little bit.

**Comment by tonyleala** [comment]
> I will run these two experiments: `res50_coco_512x512` and `higherhrnet_w32_crowdpose_512x512` in associative_embedding, and see if we can reproduce these results. @tonyleala @jin-s13

Thanks for your help！

**Comment by tonyleala** [comment]
> > The configs, work_dirs, and test results of higherhrnet_w32_crowdpose_512x512 have been uploaded below.
> > The official results cannot be reproduced. AP is about one point behind the official website results. Please help me check it, thanks! [file.zip](https://github.com/open-mmlab/mmpose/files/7272863/file.zip)
> 
> I found a small detail when I checked the log you uploaded. The steps per epoch is 104 but it should be 125. So I was wondering maybe the data you used have some missing part. You can check it and try again. Good luck!

Thanks! Related experiments are being done now.

**Comment by tonyleala** [comment]
> > The configs, work_dirs, and test results of higherhrnet_w32_crowdpose_512x512 have been uploaded below.
> > The official results cannot be reproduced. AP is about one point behind the official website results. Please help me check it, thanks! [file.zip](https://github.com/open-mmlab/mmpose/files/7272863/file.zip)
> 
> I found a small detail when I checked the log you uploaded. The steps per epoch is 104 but it should be 125. So I was wondering maybe the data you used have some missing part. You can check it and try again. Good luck!

Hello, 
It still stay the same when the 6-V100 GPUs have been used for training the model with "amples_per_gpu=8" and 209 iterations each epoch. The result of AP is still 64.5.

I haven't tried to use 8-V100 GPUs with "amples_per_gpu=24" as official. Maybe I need to keep trying untill it works.

**Comment by liqikai9** [comment]
I found that in your config file `higherhrnet_w32_crowdpose_512x512_fuxian_2.py`, on line 182, you used the **training set for training** (10000 images in total) while in the official config file we used **both the training and validation set** (12000 images in total) for training, details can be found here. 

https://github.com/open-mmlab/mmpose/blob/8d668a578f1580373d90a1eea88f23f6816df60e/configs/body/2d_kpt_sview_rgb_img/associative_embedding/crowdpose/higherhrnet_w32_crowdpose_512x512.py#L180

So, the number of images you used for training is 10000, which accounts for that when you used  6-V100 GPUs  with `samples_per_gpu=8` the number of iterations each epoch is 209 (6 * 8 * 209 = 10032 ~ 10000).

Further, I have run the experiment with the official config and 8 V-100 GPUs, and the **AP** is **0.6528**, which is just slightly slower than the released [65.5](https://github.com/open-mmlab/mmpose/blob/master/configs/body/2d_kpt_sview_rgb_img/associative_embedding/crowdpose/higherhrnet_crowdpose.md). And I think the result with a little accuracy fluctuation is acceptable.
So I suggest you modify the config file and try it again. Good luck!

**Comment by liqikai9** [comment]
> The NEW: A problem of reproducing the "res50_coco_512x512" in associative_embedding.
> 
> The experimental settings are consistent with the official, but the result of AP is about one point below the official result.
> 
> Please help me check it again, thanks!

Thanks for your error report and we appreciate it a lot.
Before you raise a possible problem, we hopefully suggest you first check the following questions which will help us locate the possible problems more effectively. 

1. What command or script did you run?
2. What config did you run?
3. Did you make any modifications on the code or config? Did you understand what you have modified?

Also, if convenient, please run `PYTHONPATH=${PWD}:$PYTHONPATH python mmpose/utils/collect_env.py` to collect necessary environment information and paste it here.

We need more detailed information about the experiment `res50_coco_512x512`  in associative_embedding you run.
